security/advancedtls/advancedtls.go

```diff
@@ -164,7 +165,7 @@ type ServerOptions struct {
 	// invoke this function every time asked to present certificates to the
 	// client when a new connection is established. This is known as peer
 	// certificate reloading.
-	GetCertificate func(*tls.ClientHelloInfo) (*tls.Certificate, error)
+	GetCertificate func(tls.ClientHelloInfo) ([]tls.Certificate, error)
```

Comment:
Let's keep the pointers to avoid unintentional copying by value.

Reply:
Sure!